rtb/DSL/encoders.hpp

```diff
@@ -13,6 +13,12 @@
 #include "jsonv/all.hpp"
 #include <boost/lexical_cast.hpp>
 
+/***** FOR BOOST ANY IMPLEMENTATION *****/
+#include <boost/any.hpp>
+#include <boost/utility/string_view.hpp>
```

Comment:
The travis build fails , looks like boost 1.6 does not have boost::string_view should I just use jsonv::string_view ?